sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -53,6 +53,7 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
       // drop all databases, tables and functions after each test
       spark.sessionState.catalog.reset()
     } finally {
+      Utils.deleteRecursively(new File(spark.sessionState.conf.warehousePath))
```

Comment:
We dropped all the database in line 54, right? Could you fix the test cases instead of delete the directory here?